lib/main_common.pm

```diff
@@ -1349,7 +1349,7 @@ sub load_yast2_ncurses_tests {
     loadtest "console/yast2_proxy";
     loadtest "console/yast2_ntpclient";
     loadtest "console/yast2_tftp";
-    loadtest "console/yast2_vnc";
+    loadtest "console/yast2_vnc" unless check_var('ARCH', 's390x');
```

Comment:
Hmm, I thought we have adjusted the module so it works on s390x for some parts, isn't it?

Reply:
it was just for opening the application but it was doing nothing...so the code looks much more clear without it.

Reply:
Yes, and it was suggested to skip it in that ticket, so let's not change our mind now when the work is done and test does work. When it breaks, we can reconsider. That would be my proposal.

Reply:
no problem, I will incorporate it again.

Reply:
Thanks, just not to feel that we have wasted time on adjustments =). But next time we should be more strict with our evaluation, as I agree that test is far from being good.

Reply:
I don't feel that, it is good that you realized that I contradict myself: before I wanted to include it for just opening the dialog and now I didn't want to cover for s390x that code because I was more concerning on maintenance(or nice code?) to avoid branches. It is great to have that feedback! ;) I will try to remark my reasoning better for next PRs. Thanks!